spec/index.html

```diff
@@ -29,21 +29,19 @@
           { name: "Chimezie Ogbuji" },
         ],
 
-        github: { repoURL: "https://github.com/w3c/sparql-entailment", branch: "main"},
+        github: "https://github.com/w3c/sparql-entailment",
```

Comment:
Remove all configuration changes.

These need to be applied to all 11 documents and it is much easier to do that on all 11 the same time with a script.

Reply:
Maybe put a comment at the top of the output that it is generated and point to where changes should be made.

Reply:
Change made.